LLama.Examples/RepoUtils.cs

```diff
@@ -1,40 +0,0 @@
-﻿using Microsoft.Extensions.Logging;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace LLama.Examples
-{
-    /// <summary>
-    /// Basic logger printing to console
-    /// </summary>
-    internal static class ConsoleLogger
```

Comment:
Not really a specific request, just thinking out loud: do you think something like this should be implemented in LLamaSharp itself?